### PR TITLE
Fix: Remove backport repo after install.

### PR DESCRIPTION
**Pull request by bjones1**
Clean: Remove unused vars, imports.

Works in production.

### CODE REVIEW

File: docker/docker_tools.py [hunk 1]
```diff
@@ -813,8 +814,6 @@ def _build_phase_2_core(
     exec(open(activate_this_path).read(), {"__file__": activate_this_path})
 
     w2p_parent = Path(env.WEB2PY_PATH).parent
-    bookserver_path = get_bookserver_path()
```
Comment (bnmnetp) [inline docker/docker_tools.py:816]: This does not seem related to the title of this PR, and not getting the bookserver_path seems concerning. What is this about?

Comment (bjones1) [reply docker/docker_tools.py:816]: This variable is never used -- I moved it to `docker_tools_misc.py`. This is cleanup from flake8.

Comment (bjones1) [reply docker/docker_tools.py:816]: (By never used, I mean that it's used only in the next line; `run_bookserver_kwargs` isn't used, making `bookserver_path` also unused).
